UI/Web/src/assets/langs/en.json

```diff
@@ -2820,12 +2820,19 @@
         "pdf-odd": "Odd",
         "pdf-even": "Even",
         "pdf-light": "Light",
-        "pdf-dark": "Dark"
+        "pdf-dark": "Dark",
+        "breakpoints": {
```

Comment:
I would prefer to move these to their own breakpoint-pipe entry and just use that.